src/main/webapp/statistics.html

```diff
@@ -231,72 +231,125 @@ <h2>Procedures</h2>
+      ['South Carolina', 20, createCustomHTMLContent(20, 'https://upload.wikimedia.org/wikipedia/commons/6/69/Flag_of_South_Carolina.svg',
+       'Illegal In the third trimester except when necessary to preserve the life or health of the pregnant woman on the written recommendation of two doctors.')],
+      ['South Dakota', 24, createCustomHTMLContent(24, 'https://upload.wikimedia.org/wikipedia/commons/1/1a/Flag_of_South_Dakota.svg',
+      'Legal only to preserve life or health of the mother in third trimester.')],
+      ['Tennessee', 24, createCustomHTMLContent(24, 'https://upload.wikimedia.org/wikipedia/commons/9/9e/Flag_of_Tennessee.svg',
+      'After viability, only if necessary to preserve life or health of mother.')],
+      ['Texas', 20, createCustomHTMLContent(20, 'https://upload.wikimedia.org/wikipedia/commons/f/f7/Flag_of_Texas.svg',
+      'Abortion during third trimester of viable child permissible only if necessary to prevent death or substantial risk of serious' +
+      'impairment to woman\'s physical or mental health, or if fetus has severe and irreversible abnormality.')],
+      ['Utah', 18, createCustomHTMLContent(18, 'https://upload.wikimedia.org/wikipedia/commons/f/f6/Flag_of_Utah.svg',
+      'individuals seeking abortions must wait 72 hours after being given informed consent of the abortion procedure to actually have the procedure done.')],
+      ['Vermont', 24, createCustomHTMLContent(24, 'https://upload.wikimedia.org/wikipedia/commons/4/49/Flag_of_Vermont.svg',
+      'none. It is legal for an abortion to be performed in Vermont at any stage of pregnancy for any reason or for no reason.')],
+      ['Virginia', 24, createCustomHTMLContent(24, 'https://upload.wikimedia.org/wikipedia/commons/4/47/Flag_of_Virginia.svg',
+      'Illegal in the third trimester, unless continuation of pregnancy likely to result in death, physical or mental impairment of mother')],
+      ['Washington', 24, createCustomHTMLContent(24, 'https://upload.wikimedia.org/wikipedia/commons/5/54/Flag_of_Washington.svg', 'none')],
+      ['West Virginia', 24, createCustomHTMLContent(24, 'https://upload.wikimedia.org/wikipedia/commons/2/22/Flag_of_West_Virginia.svg',
+      'State abortion statutes require notification of one parent prior to an abortion, and a mandatory 24-hour waiting period during which a woman must also receive counseling and a notification of services.')],
+      ['Wisconsin', 21, createCustomHTMLContent(21, 'https://upload.wikimedia.org/wikipedia/commons/2/22/Flag_of_Wisconsin.svg',
+      'All abortions are considered illegal unless necessary to save life of mother or advised by two other M.D.s as necessary')],
+      ['Wyoming', 24, createCustomHTMLContent(24,'https://upload.wikimedia.org/wikipedia/commons/b/bc/Flag_of_Wyoming.svg',
+       'Illegal after viability, unless necessary to preserve health, life of mother according to appropriate medical judgment')]
```

Comment:
I think it's fine for now! But it'll be better if you can store the data in CSV and parse it so it'll be cleaner and you can fetch the data as needed with JavaScript / Java so it won't clog up the load time at the beginning.